extensions/schema-compare/src/schemaCompareResult.ts

```diff
@@ -418,9 +421,11 @@ export class SchemaCompareResult {
 		}).component();
 
 		this.generateScriptButton.onDidClick(async (click) => {
+			Telemetry.sendTelemetryEvent('SchemaCompareGenerateScript');
```

Comment:
>SchemaCompareGenerateScript [](start = 33, length = 27)

SchemaCompareGenerateScriptStarted

Reply:
Also is there any other non-PII info you could send up? I see this operationId which seems like a likely candidate

---
In reply to: [287126270](https://github.com/microsoft/azuredatastudio/pull/5595#discussion_r287126270) [](ancestors = 287126270)

Reply:
added operationid